query.go

```diff
@@ -1078,6 +1078,8 @@ func (s *ChainService) sendTransaction(tx *wire.MsgTx, options ...QueryOption) e
 						)
 
 						numReplied++
+
+						close(peerQuit)
```

Comment:
I think this is wrong. Why would we close the connection to the peer if we just got an INV message? That means they want to know about the TX, so we send it to them. Wouldn't we also want to wait for their response before quitting?

Reply:
Closing the `peerQuit` channel is just used to indicate that the peer has responded so that we free up the resource so we can use the peer connection for other calls. Closing this channel doesnt close the connection to the peer

Reply:
Ah okay. But are we guaranteed to get further responses from that peer? E.g. if a reject message from that peer comes in later, do we get it in any case or depending on the timing (e.g. there's a chance we might not get it because we quit the peer here already)?

Reply:
I just tried this out in combination with the `lnd` test `TestLightningWallet/btcwallet/neutrino:publish_transaction`. If I remove the `close(peerQuit)` in the INV message case, I correctly get the reject for a double spent transaction. If I leave it in there, the reject is never received.
There's also a bug in the result interpretation, so I'm going to open a PR anyway. Can discuss this further in that new PR.